hugo/content/kommunen/hamburg.md

```diff
@@ -105,6 +169,17 @@ Die Daten zu den CO₂-Zielen der Stadt entsprechen dem aktuellen Klimaplan für
 
 - [Erste Fortschreibung des Hamburger Klimaplans - PDF](https://www.hamburg.de/contentblob/13287332/bc25a62e559c42bfaae795775ef1ab4e/data/d-erste-fortschreibung-hamburger-klimaplan.pdf)
 
+Transformationspfade und Maßnahmen sind Artikeln der Behörde für Umwelt und Energie entnommen
+
+- [Transformationspfade der Stadt Hamburg](https://www.hamburg.de/klimaplan/13254982/vier-transformationspfade/)
+
```

Comment:
hier sind auch Leerzeilen in der Aufzählung, s. Kommentar oben, verfahre wie beliebt :)